depends/packages/libmultiprocess.mk

```diff
@@ -3,7 +3,7 @@ $(package)_version=$(native_$(package)_version)
 $(package)_download_path=$(native_$(package)_download_path)
 $(package)_file_name=$(native_$(package)_file_name)
 $(package)_sha256_hash=$(native_$(package)_sha256_hash)
-$(package)_dependencies=native_$(package) boost capnp
+$(package)_dependencies=native_$(package) capnp
```

Comment:
But somehow it works anyhow without 19160? if I drop 24385 I'd also have to drop 24754 and 25046. I'd probably prefer to not do that if everything is functioning as expected with these changes implemented?

Reply:
by default multiprocess's build is disabled in CI because there's no any multiprocess code yet actually, only build system setup and there's nothing to test with each CI run. So, that succeed because it is not tested.

To test multiprocess you may use one of too option:
 - revert [this commit](https://github.com/dashpay/dash/pull/5815/commits/5065f206556e3de4378d4abf58101eda7f32c70a) which disabled multirpcoess in CI, check CI result and apply it back - so far as no multiprocess code yet
 - build it locally. To do that you need to clean build of depends `make MULTIPROCESS=1`; after that do clean build of code: autogen + configure with ` --enable-multiprocess` and run `BITCOIND=dash-node test/functional/test_runner.py`

https://github.com/dashpay/dash/pull/5815 - PR with multiprocess with CI enabled/disabled commits

I will enable CI when it would be any noticeable progress with multiprocess. On other hand while there's no significant progress with multiprocess maybe we can ignore if that's half broken?